distributed/dashboard/components/scheduler.py

```diff
@@ -1856,7 +1856,26 @@ def update(self):
                 )
                 continue
             try:
-                data[name].insert(0, sum(data[name]))
+                if name == "memory_percent":
+                    total_data = sum(
+                        ws.metrics["memory"] for ws in self.scheduler.workers.values()
+                    ) / sum(ws.memory_limit for ws in self.scheduler.workers.values())
+                elif name == "cpu":
+                    total_data = (
+                        sum(ws.metrics["cpu"] for ws in self.scheduler.workers.values())
+                        / 100
+                        / sum(1 for ws in self.scheduler.workers.values())
```

Comment:
Slight simplification:

```suggestion
                        / len(self.scheduler.workers.values())
```

Reply:
Good point I went too much over the top with this `sum(... for ws in self.scheduler.worker.values())`! I pushed a commit your suggestion.